x-pack/plugin/sql/jdbc/build.gradle

```diff
@@ -52,24 +52,20 @@ dependencies {
         compile (xpackProject('plugin:sql:sql-shared-client')) {
             transitive = false
         }
-        compile (xpackProject('plugin:sql:sql-proto')) {
+        compile (xpackProject('plugin:sql:sql-shared-proto')) {
             transitive = false
         }
     } else {
         bundled (xpackProject('plugin:sql:sql-shared-client')) {
             transitive = false
         }
-        bundled (xpackProject('plugin:sql:sql-proto')) {
+        bundled (xpackProject('plugin:sql:sql-shared-proto')) {
             transitive = false
         }
     }
-    compile (project(':server')) {
-        transitive = false
-    }
     compile (project(':libs:x-content')) {
         transitive = false
     }
-    compile "org.apache.lucene:lucene-core:${versions.lucene}"
     compile 'joda-time:joda-time:2.9.9'
     compile project(':libs:elasticsearch-core')
     runtime "com.fasterxml.jackson.core:jackson-core:${versions.jackson}"
```

Comment:
We'll want ot see about blasting these too in a followup. I think just removing the server dep is good enough for one PR.